src/modules/Providers/views/details/tabs/VirtualMachines/utils/filters/OvirtHostFilter.ts

```diff
@@ -7,7 +9,7 @@ const labelToFilterItem = (label: string): EnumValue =>
  * This component enables filtering the oVirt virtual machines
  * by the hostname that they are running on.
  */
-export const OvirtHostFiler = (t: (string) => string) => {
+export const ovirtHostFilter = (t: TFunction) => {
```

Comment:
i think it easier instead of adding a type simply import t directly here and omit it from args